src/styles/composeBoxStyles.js

```diff
@@ -16,4 +16,22 @@ export default {
     flex: 1,
     color: 'white',
   },
+  mentionedUserNotSubscribed: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    backgroundColor: 'rgb(230, 203, 67)',
+    paddingHorizontal: 16,
+    paddingVertical: 8,
+    borderTopWidth: 1,
+    borderTopColor: 'rgb(214, 161, 54)',
+  },
+  mentionedUserNotSubscribedText: {
+    flex: 1,
+    color: 'white',
+  },
+  mentionedUserNotSubscribedButton: {
+    backgroundColor: 'orange',
+    padding: 6,
+  },
```

Comment:
These should go in the same file as they're used -- much easier to read and understand that way.

Reply:
Moved to the component